repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/proxy/MisbehavingOriginTest.groovy

```diff
@@ -52,13 +46,7 @@ class MisbehavingOriginTest extends ReposeValveTest {
         repose.start([waitOnJmxAfterStarting: true])
     }
 
-    def cleanup() {
-        if (repose) {
-            repose.stop()
-        }
-        if (deproxy) {
-            deproxy.shutdown()
-        }
+    def cleanupSpec() {
         running = false
     }
```

Comment:
I don't remember... does Spock call the parent class `cleanupSpec()` as well, or do we need to specifically call it here?